packages/@storefront/flux-capacitor/src/core/adapters/past-purchases.ts

```diff
@@ -33,6 +34,14 @@ namespace PastPurchasesAdapter {
     return [...skus].sort(({ [field]: lhs }, { [field]: rhs }) => rhs - lhs);
   };
 
+  export const sortSkusByType = (skus: Store.PastPurchases.PastPurchaseProduct[], type?: number) => {
```

Comment:
Why is type optional? It's literally called sort skus by type and also never called without type except for in the test testing it can be called without type. pls

Reply:
As per @jmbtutor:

```
I think type should be optional and left as undefined if not given. That way, it can't accidentally collide with an enum value.
```

How do we resolve?

Reply:
@jmbtutor can you explain your reasoning? I think using a optional parameter here before it's needed is over-engineering this problem. In any case, I don't feel strongly about it so it can be included.

Reply:
Discussed IRL. Will leave in place.